package.json

```diff
@@ -31,7 +31,8 @@
     "node-zip": "^1.1.1",
     "uuid": "^3.1.0",
     "vue": "^2.3.4",
-    "xml2js": "^0.4.17"
+    "xml2js": "^0.4.17",
+    "xmlbuilder": "^9.0.1"
```

Comment:
`xml2js` depends on the `xmlbuilder` (js->xml) and `sax` (xml->js) modules. However, it uses quite old versions and I decided to use a newer version of `xmlbuilder` directly. The best way would definitely be completely dropping the `xml2js` dependencies and only using the (by the way also better maintained) sub-dependencies. For that, we also need to update the import.